### PR TITLE
moved parts to build.fish

### PR DESCRIPTION
**Pull request by fceller**

### CODE REVIEW

File: dmg/ArangoDB3e-CLI.app/Contents/Resources/arango-dfdb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```
Comment (dothebart) [inline dmg/ArangoDB3e-CLI.app/Contents/Resources/arango-dfdb]: I would vote for DRY here if possible. Pls rather have the file just once, and install it to the respective targets with a loop.

**Comment by KVS85** [comment]
- http://jenkins01.arangodb.biz:8080/job/arangodb-ANY-mac-packages/754/
- http://jenkins01.arangodb.biz:8080/job/arangodb-ANY-mac-packages/755/
- http://jenkins01.arangodb.biz:8080/job/arangodb-ANY-mac-packages/756/
- http://jenkins01.arangodb.biz:8080/job/arangodb-ANY-mac-packages/757/

**Comment by KVS85** [comment]
- 3.4: http://jenkins01.arangodb.biz:8080/job/arangodb-ANY-mac-packages/763/
- 3.5: http://jenkins01.arangodb.biz:8080/job/arangodb-ANY-mac-packages/764/
- 3.6: http://jenkins01.arangodb.biz:8080/job/arangodb-ANY-mac-packages/765/
- devel: http://jenkins01.arangodb.biz:8080/job/arangodb-ANY-mac-packages/762/

File: dmg/default/ArangoDB3e-CLI.app/Contents/Resources/arangodb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  echo "$REALPATH"
+}
+me=$(realpath $0)
+program=$(basename $me)
+installation=$(dirname $me)
+executable=$installation/opt/arangodb/bin/$program
+if [ -x $executable ] ; then
+  exec $executable --server.arangod=$installation/opt/arangodb/sbin/arangod --server.js-dir=$installation/opt/arangodb/share/arangodb3/js "$@"
+fi
+echo "Could not find executable!"
```
Comment (dothebart) [inline dmg/default/ArangoDB3e-CLI.app/Contents/Resources/arangodb]: should we print $executable here?

Comment (KVS85) [reply dmg/default/ArangoDB3e-CLI.app/Contents/Resources/arangodb]: See https://github.com/arangodb/oskar/pull/212/commits/786d8c83d11e5a15b3860c6feefae7a4dfa57631.

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/8607/
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/8608/
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/8609/
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/8610/

**Comment by KVS85** [comment]
Tests blue except unrelated failures.

**Pull request merged**